src/config.ts

```diff
@@ -135,8 +135,10 @@ class ConfigurationFactory {
       core.debug(`Loaded config: ${util.inspect(githubEvent)}`)
 
       if (githubEvent["pull_request"]) {
+        const pathNotEncoded = `${githubEvent["pull_request"]["head"]["ref"]}`.replace(/-/g, '-').replace(/\//g, '/')
+        const encodedPath = encodeURIComponent(pathNotEncoded)
```

Comment:
If you want to keep the `/` you should replace `%2F` by `/` after running the encode.
You do not need a replace statement for `-`, as it is encoded as `-`; I checked that by doing:
```
$ node
> encodeURIComponent("feature/my-branch")
'feature%2Fmy-branch'
```